### PR TITLE
alert: correct mds high cpu usage patch command

### PR DESCRIPTION
**Pull request by parth-gr**
currently if there is a horizontal scale of mds
the patch command was wrong
Corrected it using this PR

### CODE REVIEW

**Review by malayparida2000 — APPROVED** [review]
/lgtm 

This is the correct path to increase mds counts as the patches done directly to the filesystem CR will be reconciled back.

File: alerts/openshift-container-storage-operator/CephMdsCpuUsageHigh.md [hunk 1]
```diff
@@ -38,9 +38,9 @@ oc patch -n openshift-storage storagecluster ocs-storagecluster \
...
 
 ```bash
-oc patch -n openshift-storage cephfilesystem ocs-storagecluster-cephfilesystem\
+oc patch -n openshift-storage storagecluster ocs-storagecluster\
     --type merge \
-    --patch '{"spec": {"metadataServer": {"activeCount": 2}}}'
+    --patch '{"spec": {"managedResources": {"cephFilesystems":{"activeMetadataServers": 2}}}}'
 
 Make sure we have enough CPU provisioned for MDS depending on the load.
 ```
```
[the remark this answers is not in the document]
Comment (parth-gr) [reply alerts/openshift-container-storage-operator/CephMdsCpuUsageHigh.md:46]: Good catch, thanks:)

**Comment by parth-gr** [comment]
cc @jmolmo

**Comment by jmolmo** [comment]
> cc @jmolmo

Thanks for including me in the review @parth-gr 

I am focused now completely in the Ceph project, and disconnected from OCS since several moths ago

AFAIK, I think all the monitoring documentation about OCS in Openshift is stored in runbooks, for example:

https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/CephMdsCacheUsageHigh.md

I do not know if now you have changed the location for OCS alerts. I include @aruniiird and @weirdwiz to confirm if the modification should/not should to be done also/only in runbooks.

**Comment by parth-gr** [comment]
@agarwal-mudit we need to get this merge

**Review by agarwal-mudit — APPROVED** [review]
/lgtm

**Pull request merged**